aiida/orm/computers.py

```diff
@@ -297,49 +334,100 @@ def store(self):
         return super().store()
 
     @property
-    def name(self):
+    def label(self) -> str:
+        """Return the computer label.
+
+        :return: the label.
+        """
         return self._backend_entity.name
 
+    @label.setter
+    def label(self, value: str):
+        """Set the computer label.
+
+        :param value: the label to set.
+        """
+        self._backend_entity.set_name(value)
```

Comment:
interesting - why doesn't the backend entity have a setter as well?

Reply:
No particular reason other than it was never added. I haven't touched the backend yet since I was planning to do so when I would remove all deprecations and tests. Since it is the backend and so not public API I can then change whatever I need and clean things up.